docker/network-security.sh

```
@@ -0,0 +1,155 @@
+    # Consider flushing INPUT/FORWARD if you modified them
+    # iptables -P INPUT ACCEPT
+    # iptables -F INPUT
+    # iptables -P FORWARD ACCEPT
+    # iptables -F FORWARD
+    log "Network rules cleaned up."
+  fi
+}
+
+# Execute the command passed as arguments (e.g., the restricted shell)
+# Use exec to replace this script process with the target command
+log "Executing command: $@" "critical"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)